js-plugins/mcp/src/index.ts

```
@@ -0,0 +1,79 @@
+import type { JsPlugin, ResolvedUserConfig } from '@farmfe/core';
+import { FarmMcpOptions } from './types.js';
+import { setupRoutes } from './connect.js';
+import { createMcpDefaultServer } from "./server.js"
+import { searchForWorkspaceRoot } from "./search-root.js"
+import p from "picocolors"
+import fs from 'fs/promises';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)